File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.Arrays;

public class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[] arr = new int[9];
		int sum = 0;
		
		for(int i = 0; i < 9; i++) {
			arr[i] = Integer.parseInt(br.readLine());
			sum += arr[i];
		}
		
		boolean find = false;
		for(int i = 0; i < 8; i++) {
			for(int j = i + 1; j < 9; j++) {
				if(arr[i] + arr[j] == sum - 100) {
					arr[i] = 0;
					arr[j] = 0;
					find = true;
					break;
				}
			}
			if(find) {
				break;
			}
		}
		Arrays.sort(arr);
		
		for(int i = 2; i < 9; i++) {
			System.out.println(arr[i]);
		}
	}
}

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
list = []
sum = 0

for _ in range(9):
	num = int(input())
	sum += num
	list.append(num)

list.sort()

for i in range(8):
	for j in range(i + 1, 9):
		if list[i] + list[j] == sum - 100:
			for k in range(9):
				if k == i or k == j:
					continue
				else:
					print(list[k])
			exit()